docs/config.md

```diff
@@ -395,6 +395,22 @@ FSI
 
 Use the option `namePointDisplacement unused;` for solvers that do not create a pointDisplacement field, such as the RBFMeshMotionSolver.
 
+#### Restarting FSI simulations
+
+Restarting a coupled simulation using the OpenFOAM adapter works in principle in the same way as restarting a standalone OpenFOAM solver. However, the adapter and preCICE always redefine the coupling interface and the interface node locations. In case of FSI simulations, the interface deforms over time, which leads to the definition of a deformed interface during the restart. The boolean variable `resetDisplacement` (`false` by default) allows to account for the previously accumulated interface deformation such that the initial interface configuration (`t = 0`) is completely recovered. The setting here needs to coincide with the behavior of the selected solid solver.
```

Comment:
> However, the adapter and preCICE always redefine the coupling interface and the interface node locations.

What does "always" and "redefine" mean in this context? I am confused by this sentence.

> The setting here needs to coincide with the behavior of the selected solid solver.

I would use "agree" here instead of "coincide" (simpler word).

Reply:
Updated.